admin/cypress/integration/kuba_admin_spec.js

```diff
@@ -0,0 +1,48 @@
+describe("Kuba Admin Test", () => {
+  it("Visits the Kuba Admin", () => {
+    cy.visit(Cypress.env('ADMIN_URL'));
+    cy.contains('User')
+  });
+
+  it("Post user with bad email and no password", () => {
+    cy.visit(Cypress.env('ADMIN_URL'));
+    cy.get('.RaButton-label-7').contains('Create').click();
```

Comment:
The most important best practice in Cypress is to used the : https://testing-library.com/docs/cypress-testing-library/intro/

Testing Library is a bunch of selector which allow you to select elements by a11y attributes or by content.
This advice is also available for your unit tests or components tests

Also '.RaButton-label-7' is not stable because this seems to be a auto-generated classname like materialUI does.
Using className is not recommended anyway and you should use Testing Library or, if it's not possible, custom attribute or ids.